Sources/Login/ViewModels/OneLoginIntroViewModel.swift

```diff
@@ -16,14 +16,19 @@ struct OneLoginIntroViewModel: IntroViewModel, BaseViewModel {
     init(analyticsService: AnalyticsService,
          signinAction: @escaping () -> Void) {
         self.analyticsService = analyticsService
+        let event = LinkEvent(textKey: "app_signInButton",
+                              linkDomain: AppEnvironment.oneLoginBaseURL,
+                              external: .true)
```

Comment:
Given the spec on the schema for this parameter:

> Indicates wether or not a link transfers users to an external, non GDS, web domain i.e. a link to the apple app store would be "true".

I'm not sure if this would be true as we will route the user to a login page which we own.